lib/jekyll-relative-links/generator.rb

```diff
@@ -21,16 +21,52 @@ def initialize(site)
     end
 
     def generate(site)
+      # Get any configuration specified
+      # Defaults mean no change in behaviour
+      jrl_config = site.config["jekyll_relative_links"] || {}
+      @config = {}
+      @config["collections"] = jrl_config["collections"] || []
+      @config["process_all_collections"] = jrl_config["process_all_collections"] || false
+      @config["verbose"] = jrl_config["verbose"] || 0
```

Comment:
I'd make this its own memoized `config` method, merged with defaults stored in a class-level constant hash for ease of readability/maintainability.